api-specs/gem.proto

```diff
@@ -27,9 +27,9 @@ message Gem {
   string message = 2;
   double latitude = 3;
   double longitude = 4;
-  int64 creator_id = 5;
+  usersapi.User creator = 5;
```

Comment:
(mean to comment below on GemLogsWithFriend but doesn't seem possible)
Thinking if we no longer need a separate GemLogsWithFriend proto since my intention for creating it was to store the UserInfo, but it's ok let's not change haha, sorry for the protobuf churn

Reply:
I think we may still need it, except it probably doesn't need to store the `friend` field anymore. I'll change it and push directly to your branch!

Reply:
jk I didn't remvoe the friend field. I think it may be useful for displaying info about the friend / simplify things